ecogvis/functions/subFunctions.py

```diff
@@ -73,15 +73,15 @@ def __init__(self, par, htk_config=None):
         self.tbin_signal = 1 / self.fs_signal  # time bin duration [seconds]
         self.nBins = self.source.data.shape[0]     # total number of bins
         self.min_window_bins = 10                   # minimum number of bins to plot
-        self.nChTotal = self.source.data.shape[1]     # total number of channels
-        self.allChannels = np.arange(0, self.nChTotal)  # array with all channels
+        self.all_channels_ids = list(self.source.electrodes.to_dataframe().index)  # all channels ids
+        self.n_channels_total = len(self.all_channels_ids)     # total number of channels
```

Comment:
Similar issue here. See:

https://github.com/SpikeInterface/spikeextractors/pull/401/files

Also, I don't the "all_channel_ids" is an appropriate name here because we are intentionally excluding the channels that are not associated with the ElectricalSeries, so maybe `electrical_series_channel_ids` would be better